.coafile

```diff
@@ -6,3 +6,4 @@ bears = GitCommitBear
 
 [python]
 bears = PEP8Bear
+files = **.py
```

Comment:
see https://github.com/coala/coala-emacs/pull/18/commits/875adbc73d53a0f8323413d5117caec91c5c51a2 which ignores the test file from root and includes it in the test subfolder (so that `cask exec ert-runner` picks it up).  Test works, a local `coala` run from the root ignores the test file, but gitmate is still complaining.  What am I missing?  Is there a separate gitmate config?